### PR TITLE
Update stacks node configuration

### PR DESCRIPTION
**Pull request by friedger**
## Description

There are a few configuration properties of the stack node that are not described.

This PR adds missing properties and descriptions.

@agraebe

### CODE REVIEW

**Comment by friedger** [comment]
I just added the missing properties and would need some help to add the correct descriptions and examples.

File: src/pages/references/stacks-node-configuration.md [hunk 1]
```diff
@@ -233,6 +239,26 @@ Example:
 prometheus_bind = "0.0.0.0:9153"
 ```
 
+#### deny_nodes (optional)
+
+List of nodes that must not be added to neighbors
```
Comment (CharlieC3) [inline src/pages/references/stacks-node-configuration.md:244]: Comma-separated or space-separated list?

File: src/pages/references/stacks-node-configuration.md [hunk 2]
```diff
@@ -233,6 +239,26 @@ Example:
...
+
+Example:
+
+```toml
+deny_nodes = ???
```
Comment (CharlieC3) [inline src/pages/references/stacks-node-configuration.md:249]: Let's change the example to something closer to what the user might want to use.
```suggestion
deny_nodes = 1.2.3.4,exclude.this.node.org
```

File: src/pages/references/stacks-node-configuration.md [hunk 3]
```diff
@@ -223,6 +223,12 @@ Example:
 mine_microblocks = true
 ```
 
+#### microblock_frequency (optional)
```
Comment (CharlieC3) [inline src/pages/references/stacks-node-configuration.md:226]: Needs description and example to match document format.

File: src/pages/references/stacks-node-configuration.md [hunk 4]
```diff
@@ -223,6 +223,12 @@ Example:
...
+
+#### max_microblocks (optional)
```
Comment (CharlieC3) [inline src/pages/references/stacks-node-configuration.md:228]: Needs description and example to match document format.

File: src/pages/references/stacks-node-configuration.md [hunk 5]
```diff
@@ -223,6 +223,12 @@ Example:
...
+
+#### pox_sync_sample_secs (optional)
```
Comment (CharlieC3) [inline src/pages/references/stacks-node-configuration.md:230]: Needs description and example to match document format.

**Review by CharlieC3 — CHANGES_REQUESTED** [review]
Thanks for adding these! I know this is a WIP and you were probably planning on making the below changes soon. Feel free to check them off as you go.

**Comment by agraebe** [comment]
@friedger @CharlieC3 what is the next step on this PR?

**Comment by CharlieC3** [comment]
> what is the next step on this PR?

@agraebe Should be good to merge once the above comments are addressed. I'd add them myself if @friedger doesn't have the resources, but I don't know what the properties do.

**Comment by agraebe** [comment]
@friedger could you please take a look?

**Comment by agraebe** [comment]
closing due to inactivity. please reopen if/when you get a chance. thanks!

**Pull request closed without merging**